src/test/java/cucumber_runner/TestRunner.java

```diff
@@ -6,13 +6,13 @@
 import cucumber.api.junit.Cucumber;
 
 /**
- * before running any test, need to be removed #not before used tag
+ * before running any test, need to be removed #~ before used tag
  */
 @RunWith(Cucumber.class)
 @CucumberOptions(
         glue = { "stepDefinition", "hooks" },
         features = { "src/test/resources/features" },
-        tags = { "~@amiordan", "~@mraicov", "~@aciuvaga" },
+        tags = { "@amiordan", "~@mraicov", "~@aciuvaga" },
```

Comment:
why @amiordan tag is not negated and you scenarios has this tag? chose only one way

Reply:
done